developerportal/settings/base.py

```diff
@@ -354,6 +358,14 @@
 # password fields. You'll also need to enable the model backend in local settings
 USE_CONVENTIONAL_AUTH = False
 
+# django-ratelimit - see developerportal.apps.common.middleware.rate_limiter
+RATELIMIT_ENABLE = os.getenv("RATELIMIT_ENABLE", "True") == "True"
+RATELIMIT_USE_CACHE = os.getenv("RATELIMIT_USE_CACHE", "default")
+RATELIMIT_VIEW = "developerportal.apps.core.views.rate_limited"
+DEVPORTAL_RATELIMIT_DEFAULT_LIMIT = os.getenv(
```

Comment:
Off topic, but you should consider switching to `python-decouple` which is the same as `os.getenv` but it can do automatic casting and it can automatically read for a `.env` file for local development.

Reply:
💯 YES. It's already in the backlog to do and I will be v happy to get to it, if i can